chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -695,4 +695,51 @@ std::vector<Array> Split(const Array& ary, std::vector<int64_t> indices, int8_t
     return out;
 }
 
+Array Atleast2D(const Array& x) {
```

Comment:
Please rename to `AtLeast2d` ("at", "least", and "2d" are separate words).

Reply:
All in small?

Reply:
We use camel case name convention for free functions in C++. So, let's name `AtLeast2d`.

Reply:
My bad I literally missed `AtLeast2d` and saw ("at", "least", and "2d" are separate words).

Thanks